src/main/resources/static/api.yml

```diff
@@ -1415,6 +1437,21 @@ components:
       required:
         - id
         - name
+    Characteristic:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          example: 952790c0-21d7-4fd6-a7e1-9018f08d8bb0
+        name:
+          type: string
+        serviceScope:
+          type: string
+        modelScope:
+          type: string
+      required:
+        - name
```

Comment:
`id` should be required since it's always present.

Potentially also `serviceScope` and `modelScope` depending on the response to this comment: https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/175/files#r1004275092